src/vl-map-draw-point-action.js

```diff
@@ -1,18 +1,18 @@
-import {VlMapLayerAction} from '/src/vl-map-layer-action.js';
+import {VlMapDrawAction} from '/src/vl-map-draw-action.js';
 import {VlDrawAction, OlGeometryType} from '/node_modules/vl-mapactions/dist/vl-mapactions.js';
 
 /**
  * VlMapDrawPointAction
  * @class
  * @classdesc De kaart teken actie component.
  *
- * @extends VlMapLayerAction
+ * @extends VlMapDrawAction
  *
  * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-map/releases/latest|Release notes}
  * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-map/issues|Issues}
  * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-map-draw-actions.html|Demo}
  */
-export class VlMapDrawPointAction extends VlMapLayerAction {
+export class VlMapDrawPointAction extends VlMapDrawAction {
   _createAction(layer) {
     return new VlDrawAction(layer, OlGeometryType.POINT, this._callback);
```

Comment:
moet het hier ook niet new VlMapDrawPointAction(...) zijn?

Reply:
Dat is de actie die gemaakt wordt uit vl-mapactions. Zie ook de imports. Hier hoeft denk ik niet direct iets aan te veranderen.